integration_tests/tests/in_cluster.rs

```diff
@@ -118,68 +123,99 @@ impl InClusterJanusPair {
         //
         // - setting up tasks with one global aggregator and one per-account aggregator is most
         //   representative of the subscriber use cases Divvi Up supports,
-        // - pairing a global aggregator implictly marks it as "first-party" in divviup-api, which
-        //   is necessary for the task we later provision to pass a validity check.
         let paired_leader_aggregator = divviup_api
-            .pair_global_aggregator(&NewAggregatorRequest {
+            .create_shared_aggregator(NewSharedAggregator {
                 name: "leader".to_string(),
-                api_url: Self::in_cluster_aggregator_api_url(&leader_namespace).to_string(),
+                api_url: Self::in_cluster_aggregator_api_url(&leader_namespace),
                 bearer_token: leader_aggregator_api_auth_token,
+                is_first_party: true,
             })
-            .await;
+            .await
+            .unwrap();
 
         let paired_helper_aggregator = divviup_api
-            .pair_aggregator(
-                &account,
-                &NewAggregatorRequest {
+            .create_aggregator(
+                account.id,
+                NewAggregator {
                     name: "helper".to_string(),
-                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace).to_string(),
+                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace),
                     bearer_token: helper_aggregator_api_auth_token,
                 },
             )
-            .await;
+            .await
+            .unwrap();
 
+        let hpke_config = task.collector_hpke_config().unwrap();
         let collector_hpke_config = divviup_api
             .create_hpke_config(
-                &account,
-                &NewHpkeConfigRequest {
-                    name: "Integration test key".to_string(),
-                    contents: STANDARD.encode(task.collector_hpke_config().unwrap().get_encoded()),
-                },
+                account.id,
+                HpkeConfigContents::new(
+                    (hpke_config.id() as u8).into(),
+                    (hpke_config.kem_id() as u16).try_into().unwrap(),
+                    (hpke_config.kdf_id() as u16).try_into().unwrap(),
+                    (hpke_config.aead_id() as u16).try_into().unwrap(),
+                    hpke_config.public_key().as_ref().to_vec().into(),
+                ),
```

Comment:
These conversions need some fixes:
```suggestion
                &HpkeConfigContents::new(
                    u8::from(*hpke_config.id()).into(),
                    (*hpke_config.kem_id() as u16).try_into().unwrap(),
                    (*hpke_config.kdf_id() as u16).try_into().unwrap(),
                    (*hpke_config.aead_id() as u16).try_into().unwrap(),
                    hpke_config.public_key().as_ref().to_vec().into(),
                ),
```